Exiled.Events/Events.cs

```diff
@@ -101,7 +101,8 @@ public override void OnEnabled()
 
             RagdollManager.OnRagdollSpawned += Handlers.Internal.RagdollList.OnSpawnedRagdoll;
             RagdollManager.OnRagdollRemoved += Handlers.Internal.RagdollList.OnRemovedRagdoll;
-
+            ItemPickupBase.OnPickupAdded += x => Pickup.BaseToPickup.Add(x, Pickup.Get(x));
```

Comment:
oh, bruh, i forgot that `Pickup.Get(x)` includes adding to list, just keep `Pickup.Get` and remove `Pickup.BaseToPickup.Add`

Reply:
```suggestion
            ItemPickupBase.OnPickupAdded += x => Pickup.Get(x);
```